services/cognitiveservices/v1.0/customsearch/version.go

```diff
@@ -21,10 +19,10 @@ import "github.com/Azure/azure-sdk-for-go/version"
 
 // UserAgent returns the UserAgent string to use when sending http.Requests.
 func UserAgent() string {
-	return "Azure-SDK-For-Go/" + version.Number + " customsearch/1.0"
+	return "Azure-SDK-For-Go/v14.0.0 services"
```

Comment:
The UserAgent isn't formatted as we expect it to be.

Reply:
yes, because never regenerated this and reverting the breaking changes leaves it as it is. Will manually update to the central version.